evalml/tests/component_tests/test_decision_tree_classifier.py

```diff
@@ -0,0 +1,64 @@
+import numpy as np
+from sklearn.tree import DecisionTreeClassifier as SKDecisionTreeClassifier
+
+from evalml.model_family import ModelFamily
+from evalml.pipelines import DecisionTreeClassifier
+from evalml.problem_types import ProblemTypes
+
+
+def test_model_family():
+    assert DecisionTreeClassifier.model_family == ModelFamily.DECISION_TREE
+
+
+def test_problem_types():
+    assert ProblemTypes.BINARY in DecisionTreeClassifier.supported_problem_types
```

Comment:
Maybe something like this would be cleaner?
``` 
assert set(DecisionTreeClassifier.supported_problem_types) == {ProblemTypes.MULTICLASS, ProblemTypes.BINARY}
```